plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/jdbc/TrinoJdbcCatalog.java

```diff
@@ -27,6 +27,7 @@
 import io.trino.spi.connector.ConnectorViewDefinition;
```

Comment:
Commit comment:
`Support registering and unregistering tables in Iceberg JDBC` -> `Support registering and unregistering tables in Iceberg JDBC catalog`